easybuild/easyconfigs/b/BuildEnv/BuildEnv-1.0-intel-2015a-default.eb

```diff
@@ -0,0 +1,13 @@
+easyblock = "BuildEnv"
```

Comment:
I'd prefer to go the other way and rename the easyblock then, including eb in the name introduces a layer of obscurity for those who don't know what eb stands for (and they're the ones this is intended for).